### PR TITLE
added support for new fonts, minor style changes

### PR DESCRIPTION
**Pull request by aravwatwani**
Hey Joseph! Here is what I achieved with this commit:

- Now allowed support for new fonts not built into React Native (in case stock React Native font eventually needs to be changed)
- Minor style changes, including cleaning up search bars, improved error messages when scanning items that have incorrect barcodes or currently are not carried, and fixed the header to match style guide on Figma

I was able to rename a few variables to match the style guide in Figma, but I need to continue to do this, as well as reflect these variable names across all files. I've left a <a href="https://github.com/josephycyeh/attain-web/commit/1469ca8284285620a1fe668a68990a7cee087e28?diff=unified#r73984976">comment</a> where you can see some examples of refactoring.

Let me know how this is!

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -82,12 +83,14 @@ export default function App() {
 function AppComponent() {
   
 
+  let [fontsLoaded] = useFonts({
```
Comment (josephycyeh) [inline App.js:86]: const? maybe

File: screens/homeScreen/homeScreen.js [hunk 2]
```diff
@@ -92,13 +95,13 @@ const styles = StyleSheet.create({
...
     paddingRight:20
   },
-  boldText: {
-    fontSize: 25,
+  h1FontSemiBold: {
+    fontSize: 24,
     fontWeight: "600"
   },
-  boldSecondaryText: {
+  h3FontRegular: {
```
Comment (josephycyeh) [inline screens/homeScreen/homeScreen.js:102]: I'm assuming "Font" here just means the default font?

File: App.js [hunk 3]
```diff
@@ -82,12 +83,14 @@ export default function App() {
...
+    'Lato': require('./assets/fonts/Lato.ttf'),
+    'Inter': require('./assets/fonts/Inter.otf')
+  });
```
Comment (josephycyeh) [inline App.js:89]: are we using these fonts anywhere right now?

**Review by josephycyeh — COMMENTED** [review]
overall lgtm! for this one it's ok, but for more involved changes, be sure to include screenshots

File: screens/homeScreen/homeScreen.js (hunk 2 above)
Comment (aravwatwani) [reply screens/homeScreen/homeScreen.js:102]: yeah font is default font. on the figma file the naming convention is "H3 Lato Regular" but we aren't using Lato so I used "Font" instead. if you need me to change this for clarity lmk.

File: App.js (hunk 3 above)
Comment (aravwatwani) [reply App.js:89]: not right now, no

**Pull request closed without merging**